gandalf.js

```diff
@@ -9,35 +9,33 @@
  * Copyright 2016 Henrik Ingo (@henrikingo)
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.